MatrixKit/Views/MXKImageView.m

```diff
@@ -55,6 +55,9 @@ @interface MXKImageView ()
     
     // Subviews
     UIScrollView *scrollView;
+
+    // Current attachment being displayed in the MXKImageView
+    MXKAttachment *currentAttachment;
```

Comment:
Note: `currentAttachment` must be reseted on [setImageURI: withType: andImageOrientation: isThumbnail:: previewImage: mediaManager:] call.
Suggestion: reset `currentAttachment` on [setImage:] call.